nginx_vts_exporter.go

```diff
@@ -340,16 +350,17 @@ func fetchHTTP(uri string, timeout time.Duration) func() (io.ReadCloser, error)
 }
 
 var (
-	listenAddress   = flag.String("telemetry.address", ":9113", "Address on which to expose metrics.")
+	listenAddress   = flag.String("telemetry.address", ":9913", "Address on which to expose metrics.")
```

Comment:
It was just a random port number, it is configurable with **METRICS_ADDR** Env variable if you run it inside a container.